src/goMain.ts

```diff
@@ -117,6 +118,10 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		removeTags(args);
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.impl.cursor', () => {
+		implCursor();
```

Comment:
You need to add this command in `package.json` as well, so that the command shows up in the command pallet.

Am curious, how did you test this without the command showing up in the command pallet?